modules/setup/library/Setup/ModuleDependency.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Icinga\Module\Setup;
+
+use Icinga\Application\Modules\Module;
+use Icinga\Module\Setup\Requirement\ModuleMissingRequirement;
+use Icinga\Module\Setup\Requirement\SetRequirement;
+use Icinga\Module\Setup\Requirement\WebModuleRequirement;
+
+class ModuleDependency
```

Comment:
How does this work? It's not a page nor a wizard.

Reply:
`setup-requirements.phtml` (line 12) gets the wizard and calls `getRequirements()`.
The class `ModuleDependency` contains only this method, i.e. if no `SetupWizard` is provided for the current module (e.g. Cube), this class returns module and lib dependencies.